scalameta/contrib/src/main/scala/scala/meta/contrib/implicits/implicits.scala

```diff
@@ -0,0 +1,3 @@
+package scala.meta.contrib.implicits
+
+trait implicits extends Statements.ToStatementOps
```

Comment:
The idea i general is that the use can import instances, without the implicit parts.

eg. the user can import only instances then write:

```Statements[Defn.Val].getStats(someVal)```

or create pimps dependant on `Statements` without importing all the imports.
note that in this case I have still injected both instances and implicits into the package object. its purely there for an a la carte importing similar to how Cats does it.

I personally will always use only `import scala.meta.contrib._`

I'm quite happy with whatever is decided in terms of this pattern